README.md

```diff
@@ -6,6 +6,15 @@
 
 Python client for the [kubernetes](http://kubernetes.io/) API.
 
+## Requirements
+
+A version of python with OpenSSL version 1.0.0 or later.
```

Comment:
I am not sure if this is the minimum requirement. Worth mentioning this is on mac. What we know is, the client does not work with current OPENSSL_VERSION default on osx.